system_files/desktop/usr/share/ublue-os/just/custom.just

```diff
@@ -96,6 +96,9 @@ resize-zram:
     just --unstable resize-zram
   fi
 
+set-btrfs-flags:
+  sudo sed -i 's/compress=zstd:1/defaults,nofail,x-systemd.growfs,noatime,lazytime,commit=120,compress-force=zstd:1,space_cache=v2,discard=async/g' /etc/fstab
```

Comment:
Good for SteamOS, but we don't need some of these. I recommend:
`sudo sed -i 's/compress=zstd:1/noatime,lazytime,commit=120,compress-force=zstd:1,space_cache=v2,discard=async/g' /etc/fstab`

Reply:
Awesome. Sounds good to me. I'll adjust them